modules/tclap/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://github.com/mirror/tclap",
+    "maintainers": [
+        {
+            "github": "DAarno",
```

Comment:
Wait, will @DAarno actually help maintain this module? This should be the module maintainer for BCR, it doesn't have to be the maintainer of the original project.